src/pages/ws/ui/control-panel/zone-modals/components/zone-item-portal-wrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement, useEffect } from 'react'
```

Comment:
What's the point of creating this component, if you could do the same stuff inside PopperTableModal, since it's used only for zones in table? And then wrap it just with <Portal>

Reply:
Moved it